source/mainnet/net/web3-id/issuer.rst

```diff
@@ -0,0 +1,144 @@
+.. _web3id-issuer:
+.. include:: ../../variables.rst
+
+.. meta::
+    :description lang=en:
+        Anyone can become an issuer of verifiable credentials for Web3 ID. Concordium provides an easy issuer tool to start the process.
+
+===============
+Web3 ID issuers
+===============
+
+Web3 ID includes many types of credentials that are not identity credentials. This might be employment history, certifications from educational courses or diplomas, membership in loyalty clubs or rewards programs, and more. Because of this anyone can become an issuer!
+
+How to become an issuer
+=======================
+
+An issuer needs to have the following components.
+
+1. Some way of identifying users to whom the credentials will be issued, e.g., an existing database of users.
+2. JSON schemas and metadata for credentials. These are used by the wallet to display and style the credential. This might be a logo, a background color, how the attributes will appear in the credential (e.g., date format, title), and so on.
+3. A smart contract where credential lifetime is managed, together with its metadata such as expiry time.
```

Comment:
Could we also state it here that the issuer onboarding page helps with setting up these two.